src/Microsoft.VisualStudio.SlowCheetah.VS/NugetHandler/PackageHandlers/EmptyHandler.cs

```diff
@@ -15,18 +17,19 @@ internal class EmptyHandler : IPackageHandler
         /// Initializes a new instance of the <see cref="EmptyHandler"/> class.
         /// </summary>
         /// <param name="package">VS package</param>
-        public EmptyHandler(IServiceProvider package)
+        public EmptyHandler(AsyncPackage package)
         {
             this.Package = package ?? throw new ArgumentNullException(nameof(package));
         }
 
         /// <inheritdoc/>
-        public IServiceProvider Package { get; }
+        public AsyncPackage Package { get; }
 
         /// <inheritdoc/>
-        public void Execute(Project project)
+        public TPL.Task Execute(Project project)
         {
             // Do nothing
+            return TPL.Task.FromResult(false);
```

Comment:
Do you not have `Task.CompletedTask;`? If it isn't in the BCL (I think it is net4.6.1, so you might now have it), VS-threading should have one in it.